test-integration/helpers/test-element.coffee

```diff
@@ -51,7 +51,8 @@ Object.defineProperties TestItemHelper.prototype,
 
 
 class TestHelper extends TestItemHelper
-  constructor: (test, testElementLocator, commonElements = {}, options) ->
+  constructor: (test, testElementLocator, commonElements, options) ->
+    commonElements ||= _.result(@, 'elementRefs', {})
     defaultOptions =
```

Comment:
@pandafulmanda (and rest of FE team) what do you think about this?  It attempts to read the commonElements from an `elementRefs` property of the object.  I like setting them there better than an unattached constant from outside the class